pom.xml

```diff
@@ -124,25 +124,25 @@
 
   <properties>
     <amqp-10-starter.version>2.2.2</amqp-10-starter.version>
-    <dekorate.version>1.0.0</dekorate.version>
+    <dekorate.version>1.0.2</dekorate.version>
     <infinispan-starter.version>2.2.3.Final</infinispan-starter.version>
-    <keycloak.version>9.0.3</keycloak.version>
-    <narayana-starter.version>2.3.2</narayana-starter.version>
+    <keycloak.version>9.0.9</keycloak.version>
```

Comment:
From the jira ticket we are waiting for the upstream version for keycloak but this one is aligned with the downstream one

Reply:
You're right, although I asked for it I guess I updated before having the answer.